src/_userflair.scss

```diff
@@ -16,108 +16,9 @@
 	overflow: visible !important;
```

Comment:
This line should be removed to avoid the flair icons disrespecting the border radius of the flair itself.